hack/setup-management/main.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/kelseyhightower/envconfig"
 	"github.com/sirupsen/logrus"
 
+	armconst "github.com/openshift/openshift-azure/pkg/arm/const"
+	fakerparm "github.com/openshift/openshift-azure/pkg/fakerp/arm"
+	farmconst "github.com/openshift/openshift-azure/pkg/fakerp/arm/const"
```

Comment:
@jim-minter  this is why I dont want to split constantas to fakeRP and production. tone of ARM pacakges now :/

Reply:
too bad!